### DISCUSSION TITLE
Particles should not despawn immediately

### DISCUSSION THREAD

**Discussion by Jochengehtab** [Minestom/Minestom#1755] [Help] [answered]
Hey, i'm experimenting with particles and would like to know how I should do it so that the particles don't despawn immediately.

**Comment by emortaldev** [comment, marked as answer]
Unfortunately the particle lifetime depends on the particle type and its settings. For example, the bubble pop particle has 1 tick of lifetime versus redstone dust which depends on the scale parameter for how long it appears. (This is a client limitation)

However you can simply re-spawn particles at the same point to simulate a longer lifetime

**Comment by Jochengehtab** [comment]
Ok thank you for the tip
